# Comments provide naming from Technical Customer Information

std_msgs/Header header
LocationAttributesHeader location_attributes_header
LocationAttributesPacket location_attributes_packet
# LocAtr_MountingPosition
SensorMounting mounting_position
